ocaml/xapi/xapi_sr.ml

```diff
@@ -29,6 +29,12 @@ open Xapi_database.Db_filter_types
 open API
 open Client
 
+module RefSet = Set.Make (struct
```

Comment:
I would say rather than putting a `RefSet` here at the top of this module, I would recommned using first class modules to encapsulate this inside the `scan` function as it is currently the only user of this `RefSet` module